### ISSUE TITLE
Filtering fields ignored inside fragments

### ISSUE THREAD

**Issue by felix-zg** [graphql-python/graphene-django#21]
Hello guys,

I'm trying to run this query and it's completely ignore the 'employeeType'(or any other) filter:

```
{
 business(id: "") {
    ...F2
  }
}

fragment F2 on BusinessNode {
  employee(employeeType:":ceo") {
    edges {
      node {
        status
        employeeType
        id
      }
    }
  }
}
```

On the second hand, this query works perfectly fine:

```
employee(employeeType:":ceo") {
    edges {
     node {
       status
       employeeType
       id
      }
    }
  }
```

Both of the objects(BusinessNode and EmployeeNode) are DjangoObjectType.

Thanks,
Felix

**Comment by KaRi94** [comment]
Same problem +1

**Comment by syrusakbary** [comment]
That's a weird issue. Maybe is a bug in `graphql-core` or `graphene` itself.

Could you post a fully working example that raises the issue? (a gist or a PR with a failing test would also work :) )

**Comment by felix-zg** [comment]
I used the cookbook example, and it's works perfectly fine:

```
 {
  allIngredients(notes: "Comes from a cow") {
    edges {
      node {
        id
        notes
        name
      }
    }
  }
}
```

but with fragment:
```
{
  allCategories(name: "Dairy") {
    edges {
      node {
        ...F2
      }
    }
  }
}

fragment F2 on CategoryNode {
  ingredients(notes: "Comes from a cow") {
    edges {
      node {
        id
        notes
        name
      }
    }
  }
}
```

**Comment by BossGrand** [comment]
Seeing the same issue

**Comment by BossGrand** [comment]
It doesn't even need to be a fragment for this to happen. If you run this query on the cookbook example graphiql page you can see the bug. The name filter is completely ignored 
    

  query{
      allCategories {
        edges {
           node {
               id,
               ingredients(name_Icontains:"beef") {
                  edges {
                     node {
                        id,
                        name
                     }
                  }
               }
            }
         }
      }
   }

**Comment by tom-zeit** [comment]
This issue prevents us from upgrading to graphene 1.0 :/ 
Anyway has a lead on how to solve it?

**Comment by timothyjlaurent** [comment]
I'm just noticing this too after writing tests for our app migration... This is a real problem... Need to be able to to filter on nested nodes. 

```
            query ($docid: String!, $label: String!) {
                allGene(docid_In: $docid) {  <-- this filter works
                    edges {
                        node {
                            id
                            docid
                            geneReportText (label: $label) { <-- this filter doesn't do anything
                                edges {
                                    node {
                                        text
                                        label
                                        status
                                    }
                                }
                            }
                        }
                    }
                }
            }
```

Please point me to where I can fix it @syrusakbary

**Comment by timothyjlaurent** [comment]
This was solved in #30 by @drakon

If you would like to try it 

pip install this:
```
-e git://github.com/timothyjlaurent/graphene-django.git@94ab3d6d593eb514fad97132870fc99bbdca245d#egg=graphene-django
```

**Comment by syrusakbary** [comment]
This should be fixed with #82. Closing the issue
